### ISSUE TITLE
Removal Candidate

### ISSUE THREAD

**Issue by Adurol** [KaOS-Community-Packages/itch#1]
This Package will be removed in 3 days from KCP.

Reason: **[Blind copy](https://chakraos.org/ccr/pkgbuild_view.php?p=itch-bin)**
Only submit packages that were adjusted for KaOS or you created yourself for KaOS. Blindly copying any existing PKGBUILD and submitting it to KCP is not the reason it was created for. If you can find working PKGBUILDs elsewhere, use them.

Please see also: [Using KaOS Community Packages](https://forum.kaosx.us/d/145-using-kaos-community-packages) and [KCP Guide](https://kaosx.us/docs/kcp/#submitting-a-package)

**Comment by sixsixfive** [comment]
so copying myself is a blind copy? well, then...

**Comment by Adurol** [comment]
No, a blind copy is it if the package submitted to KCP can be found almost identical by similar providers. To say it in another way, if the PKGBUILD found for example on AUR and it just works for KaOS, no need to to upload it to KCP, regardless who the "creator" was. In your case, it just happened that you uploaded the PKGBUILD you submitted to CCR also to KCP. If someone else had uploaded a blind copy of itch from CCR, AUR, Manjaro, ... I would have opened the same issue. Please note, that this is not against you personal, it's just how things are done on KCP.

Granted, what could be counted as blind copy is debatable.
